libbeat/tests/system/requirements_aix.txt

```diff
@@ -49,3 +50,4 @@ urllib3==1.26.18
 wcwidth==0.2.5
 websocket-client==0.47.0
 zipp>=1.2.0,<=3.1.0
+protobuf==3.19.4 #Temporary change because of protobuf new version bug: https://github.com/protocolbuffers/protobuf/issues/10051
```

Comment:
this is not part of my fix!?

Reply:
I coped the non-aix version over the top of the aix version and it picked this up.

Really we should just remove these aix requirements files since we don't support the aix architecture or properly maintain this.